terraform/vars/terraform-dev.tfvars

```diff
@@ -4,3 +4,6 @@ tile_cache_url            = "dev-tiles.globalforestwatch.org"
 desired_count             = 1
 auto_scaling_min_capacity = 1
 auto_scaling_max_capacity = 5
+fargate_cpu               = 4096
+fargate_memory            = 8192
+data_lake_bucket_name     = "gfw-data-lake-staging"
```

Comment:
Is this correct for dev tfvars, or should it be "gfw-data-lake-dev" ?

Reply:
Yes, the dev account doesn't have much data to work with and this is a read only access to the assets in staging. We are having to deploy our code changes to staging before it goes thru review for that reason right now. This is a one step forward to being able to use staging assets without deploying untested code there.